src/libraries/System.Linq.Parallel/src/System.Linq.Parallel.csproj

```diff
@@ -1,13 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+    <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppCurrent)-browser</TargetFrameworks>
```

Comment:
Isn't the Browser TFM only necessary when the MonoWasmBuildVariant switch is passed is? Otherwise the TFMs would be identical.

Reply:
Yes, it is. There was discussion about it yesterday in https://github.com/dotnet/runtime/pull/97505#discussion_r1467081947.

This particular change is to get rid of TargetOS check and make it behave in the same way as the other library projects.

Reply:
```suggestion
    <TargetFramework Condition="'$(MonoWasmBuildVariant)' != 'multithread'">$(NetCoreAppCurrent)</TargetFramework>
    <TargetFrameworks Condition="'$(MonoWasmBuildVariant)' == 'multithread'">$(NetCoreAppCurrent);$(NetCoreAppCurrent)-browser</TargetFrameworks>
```

Reply:
Please make the same change in the other project that you updated in the other PR.

Reply:
We probably don't need to build `$(NetCoreAppCurrent)` in the browser case though

Reply:
All libraries need a TFM agnostic TFM in their list, otherwise NuGet restore will fail.

Reply:
Is Browser ever built without `MonoWasmBuildVariant=multithread`?

Reply:
> Is the shipping product for Browser ever built without `MonoWasmBuildVariant=multithread`?

Yes, the single threaded version is still the default one.